spec/registrant_spec.rb

```diff
@@ -0,0 +1,66 @@
+require './lib/registrant'
+
+RSpec.describe @registrant do
+    before(:all) do
+    end
+
+    it 'is a registrant' do
+        registrant_1 = Registrant.new('Bruce', 18, true, "license") 
+    end
+
+    # it 'has a name' do
+    #   @name = name
```

Comment:
We don't have access to the `@name` variable in the spec file. We'd need to test this by first instantiating a new Registrant object on line 12 (like you have on line 8), then we can test the method of `.name`